src/components/Header/Header.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useEffect } from 'react';
+import cn from 'classnames';
+
+import { Todo } from '../../types/Todo';
+import { USER_ID } from '../../api/todos';
+import { showErrorMesage } from '../../utils/showErrorMesage';
+
+type TodosState = {
+  todos: Todo[];
+  setTodos: (el: Todo[]) => void;
+};
+
+type TitleState = {
+  todoTitle: string;
+  setTodoTitle: (el: string) => void;
+};
+
+type Props = {
+  todosState: TodosState;
+  titleState: TitleState;
+  reset: () => void;
+  inputRef: React.RefObject<HTMLInputElement>;
+  isDeleting: boolean;
+  loading: boolean;
+  addTodos: (
+    title: string,
+    completed: boolean,
+    userId: number,
+  ) => Promise<Todo | void>;
+  setErrorMessage: (el: string) => void;
+};
+
+export const Header: React.FC<Props> = ({
+  todosState,
+  titleState,
+  reset,
+  inputRef,
+  isDeleting,
+  loading,
+  addTodos,
+  setErrorMessage,
+}) => {
+  //#region Setvice Functions
+
+  //#endregion
+
+  useEffect(() => {
+    if (!loading && inputRef.current) {
+      inputRef.current.focus();
+    }
+  }, [loading]);
```

Comment:
Fit it or remove eslint fo this line 
<img width="968" alt="image" src="https://github.com/user-attachments/assets/35bf097f-c1f1-449c-b454-c670665a2fdb">